training/frameworks/pytorch/lightning/single_node_single_device_mnist.ipynb

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
<p>Add a title to the notebook, <code># Title</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3522/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>